.github/workflows/ci.yml

```
@@ -231,6 +231,45 @@ jobs:
       - name: Run yarn test262
         run: exit 0 # TODO remove test262 from required tests for CI
 
+  test-hermes:
+    name: test-hermes
+
+    # begin macro
+
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
+        platform: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v4
```

like we've started using in

https://github.com/endojs/endo/blob/79e90ebae1661241d5cfdaf6417055e8baef6a59/.github/workflows/typedoc-gh-pages.yml#L26